integration/screen_widgets_test.go

```diff
@@ -4,7 +4,6 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
-	"github.com/zorkian/go-datadog-api"
```

Comment:
Don't remove this line, as it's needed to pull in types:
```
[go-datadog-api] make testacc                                           11:51:09  ☁  dabcoder-master ☂ ✭
go test integration/* -v  -timeout 90m
# command-line-arguments [command-line-arguments.test]
integration/screen_widgets_test.go:10:15: undefined: datadog
FAIL	command-line-arguments [build failed]
make: *** [Makefile:17: testacc] Error 2
```

Reply:
Not sure what happened there, didn't mean to remove it, but good catch, re-added it.